### PR TITLE
[Reviewer: Ellie] Add Memcached plugin

### PR DESCRIPTION
**Pull request by rkd-msw**

### CODE REVIEW

File: memento.root/usr/share/clearwater/clearwater-cluster-manager/plugins/memento_memcached_plugin.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Project Clearwater - IMS in the Cloud
```
Comment (eleanor-merry) [inline memento.root/usr/share/clearwater/clearwater-cluster-manager/plugins/memento_memcached_plugin.py]: Name of the file?

File: memento.root/usr/share/clearwater/clearwater-cluster-manager/plugins/memento_memcached_plugin.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+
+class MementoMemcachedPlugin(SynchroniserPluginBase):
+    def __init__(self, _ip):
+        issue_alarm(constants.RAISE_MEMCACHED_NOT_YET_CLUSTERED)
+
+    def key(self):
+        return "/memento/clustering/memcached"
+
+    def files(self):
```
Comment (eleanor-merry) [inline memento.root/usr/share/clearwater/clearwater-cluster-manager/plugins/memento_memcached_plugin.py]: This is new. Do I need to update the Chronos plugins with this as well?

**Comment by eleanor-merry** [comment]
Looks good, two comments. 

Also, one of these pull requests should reload the cluster manager after the install - do you have one for this?

File: memento.root/usr/share/clearwater/clearwater-cluster-manager/plugins/memento_memcached_plugin.py (hunk 2 above)
Comment (rkd-msw) [inline memento.root/usr/share/clearwater/clearwater-cluster-manager/plugins/memento_memcached_plugin.py]: This is the "only load plugins that don't try to edit the same file" stuff we discussed - IIRC I added it to the Sprout plugin after you checked it in, and that's copied to the Ralf one (https://github.com/Metaswitch/ralf/pull/147/files#diff-7c81eccccc329959ba73eaf43456e165R56).

Comment (eleanor-merry) [inline memento.root/usr/share/clearwater/clearwater-cluster-manager/plugins/memento_memcached_plugin.py]: Cool

**Pull request merged**